### PR TITLE
[RA2] Ch8 - Gap documentation template

### LINKED ISSUE #985: [RA2] Ch8 - document how to define a gap
From deep dive in Prague:
A gap should refer to a requirement in RA#2 or in the RM to allow for tracability. --> PW to create a template entry howto document a gap.

### PR DESCRIPTION
**Pull request by peterwoerndle**
Proposing a template to document gaps and enable traceability back to requirements in RA and RM. This PR addresses Fixes #985.

### CODE REVIEW

**Comment by tomkivlin** [comment]
@rabi-abdel to add Ramki as reviewer

**Comment by tomkivlin** [comment]
@ramkri123 are you able to approve please?  It will be ready for @rabi-abdel to merge then.

File: doc/ref_arch/kubernetes/chapters/chapter08.md [hunk 1]
```diff
@@ -20,6 +20,15 @@ While this Reference Architecture is being developed, Gaps will be identified th
 - Multi-Tenancy
 - K8s as VM based VNF Orchestrator
 
+<a name="8.2.0"></a>
+### 8.2.0 Gap template
+
+> **Related requirements:** List the requirement references `req.abc.xyz.00` from RA2 or RM which this gap tries to address.
+
+> **Baseline project:** Describe against which upstream project and which version of the project the gap exisits e.g. _Kubernetes v1.17_. If the gap is not against any specific upstream project, state "none".
```
Comment (deebhatia) [inline doc/ref_arch/kubernetes/chapters/chapter08.md]: nit: exisits -> exists

Comment (rabiabdel) [reply doc/ref_arch/kubernetes/chapters/chapter08.md]: Fixed in behalf of @peterwoerndle

**Pull request merged**